vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/GeneratedVaadinComboBox.java

```diff
@@ -1259,220 +101,21 @@ public String getFilter() {
-        getElement().getChildren()
-                .forEach(child -> child.removeAttribute("slot"));
-        getElement().removeAllChildren();
-    }
-
-    /**
-     * Constructs a new GeneratedVaadinComboBox component with the given
-     * arguments.
-     *
-     * @param initialValue
-     *            the initial value to set to the value
-     * @param defaultValue
-     *            the default value to use if the value isn't defined
-     * @param elementPropertyType
-     *            the type of the element property
-     * @param presentationToModel
-     *            a function that converts a string value to a model value
-     * @param modelToPresentation
-     *            a function that converts a model value to a string value
-     * @param <P>
-     *            the property type
-     */
-    public <P> GeneratedVaadinComboBox(T initialValue, T defaultValue,
-            Class<P> elementPropertyType,
-            SerializableFunction<P, T> presentationToModel,
-            SerializableFunction<T, P> modelToPresentation) {
-        super("value", defaultValue, elementPropertyType, presentationToModel,
-                modelToPresentation);
-        if (initialValue != null) {
-            setPresentationValue(initialValue);
-        }
-    }
-
-    /**
-     * Constructs a new GeneratedVaadinComboBox component with the given
-     * arguments.
-     *
-     * @param initialValue
-     *            the initial value to set to the value
-     * @param defaultValue
-     *            the default value to use if the value isn't defined
-     * @param acceptNullValues
-     *            whether <code>null</code> is accepted as a model value
-     */
-    public GeneratedVaadinComboBox(T initialValue, T defaultValue,
-            boolean acceptNullValues) {
-        super("value", defaultValue, acceptNullValues);
-        if (initialValue != null) {
-            setPresentationValue(initialValue);
-        }
-    }
-
-    /**
-     * Constructs a new GeneratedVaadinComboBox component with the given
-     * arguments.
-     *
-     * @param initialValue
-     *            the initial value to set to the value
-     * @param defaultValue
-     *            the default value to use if the value isn't defined
-     * @param elementPropertyType
-     *            the type of the element property
-     * @param presentationToModel
-     *            a function that accepts this component and a property value
-     *            and returns a model value
-     * @param modelToPresentation
-     *            a function that accepts this component and a model value and
-     *            returns a property value
-     * @param isInitialValueOptional
-     *            if {@code isInitialValueOptional} is {@code true} then the
-     *            initial value is used only if element has no {@code "value"}
-     *            property value, otherwise element {@code "value"} property is
-     *            ignored and the initial value is set
```

Comment:
Just to clarify, is this logic added by https://github.com/vaadin/vaadin-combo-box-flow/pull/394 no longer necessary?

Reply:
From my understanding not. It seems this construct for initializing the presentation value was auto-generated for all field components. However for ComboBox the initial value was hard-coded to `null`, so the condition below could never pass.
Besides the presentation value for an empty/null value in `ComboBox` is `null` as well, so there is no need to initialize the presentation value otherwise.

Reply:
Thanks, that makes sense 👍